src/io.ts

```diff
@@ -113,19 +129,21 @@ export interface AoIORead {
   }: {
     address: WalletAddress;
   }): Promise<AoGateway | undefined>;
-  getGateways(): Promise<
-    Record<WalletAddress, AoGateway> | Record<string, never>
-  >;
+  getGateways(
+    params?: PaginationParams,
+  ): Promise<PaginationResult<AoGateway & { gatewayAddress: WalletAddress }>>;
```

Comment:
Would be nice to have `AoGateway & { gatewayAddress: WalletAddress}` be a named type. Otherwise it bubbles out through the client's codebase and gets a bit verbose.

Reply:
[2db664a](https://github.com/ar-io/ar-io-sdk/pull/138/commits/2db664a4310837e05e25e5206f304925ec911d60)